steps/wetransfer-upload/v0.0.2/step.yml

```diff
@@ -0,0 +1,52 @@
+title: wetransfer-upload
+summary: |
+  Upload files to wetransfer and share a short url to download
+description: |
+  This step send files to wetransfer. Wetransfer give you a short url to download files. Limit 2GO.
```

Comment:
Please describe more what this step is supposed to do, and what setup is required for using this step.